shippy-measurement/analysis/analysis.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
```

Comment:
Now that you started with require, maybe we could even set up the whole Shippy project with a proper require structure. But maybe it's also not worth it now :)

Reply:
Oh. I'm confused. Is this actually a backend? Maybe let's talk about it :)